h5t.go

```diff
@@ -109,13 +109,12 @@ func OpenDatatype(c CommonFG, name string, tapl_id int) (*Datatype, error) {
 // NewDatatype creates a Datatype from an hdf5 id.
 func NewDatatype(id C.hid_t) *Datatype {
 	t := &Datatype{Identifier{id}}
-	runtime.SetFinalizer(t, (*Datatype).finalizer)
 	return t
 }
 
-// CreateDatatype creates a new datatype.
-// class must be T_COMPUND, T_OPAQUE, T_ENUM or T_STRING.
-// size is the size of the new datatype in bytes.
+// CreateDatatype creates a new datatype. The value of class must be T_COMPUND,
```

Comment:
actually, it's `s/T_COMPUND/T_COMPOUND/` :)

Reply:
Fixed